README.md

```diff
@@ -98,154 +98,112 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Fixed exception occurring on theme change.
+ - Fix "What's new" icon. Thanks [@niels9001](https://github.com/niels9001)!
+ - Remove obsolete UI Font icon properties. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+ - OOBE UI improvements. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+ - XAML Binding improvements. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
+ - Fixed crash caused by ThemeListener constructor exceptions.
 
 ### Documentation
 
- - Fixed some typos in the README. Thanks [@Asymtode712](https://github.com/Asymtode712)!
- - Reworked the gpo docs on learn.microsoft.com, adding .admx, registry and Intune information. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Improved docs for adding new languages to monaco. Thanks [@PesBandi](https://github.com/PesBandi)!
+ - Update README.md to directly state x64 & ARM processor in requirements.
+ - Added Scoop plugin to PowerToys Run thirdPartyRunPlugins.md docs. Thanks [@Quriz](https://github.com/Quriz)!
 
 ### Development
 
- - Updated the check-spelling ci action to 0.22. Thanks [@jsoref](https://github.com/jsoref)!
- - Refactored the modules data model used between the Settings Dashboard and Flyout.
- - Fixed a flaky interop test that was causing automated CI to hang occasionally.
- - Increased the WebView 2 loading timeout to reduce flakiness in those tests. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Added support for building with the Dev Drive CopyOnWrite feature, increasing build speed. Thanks [@pedrolamas](https://github.com/pedrolamas)!
- - Addressed the C# static analyzers suggestions. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Addressed the C++ static analyzers suggestions.
- - PRs that only contain Markdown or text files changes no longer trigger the full CI. Thanks [@snickler](https://github.com/snickler)!
- - Updated the Microsoft.Windows.CsWinRT to 2.0.4 to fix building with the official Visual Studio 17.8 release.
- - Fixed new code quality issues caught by the official Visual Studio 17.8 release.
- - Added a bot trigger to point contributors to the main new contribution issue on GitHub. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Removed unneeded entries from expect.txt.
- - Turned off a new feature from Visual Studio that was adding the commit hash to the binary files Product Version.
- - Refactored and reviewed the spellcheck entries into different files. Thanks [@Jay-o-Way](https://github.com/Jay-o-Way)!
- - Added Spectre mitigation and SHA256 hash creation for some DLLs.
- - Reverted the release pipeline template to a previous release that's stable for shipping PowerToys.
-
-#### What is being planned for version 0.77
-
-For [v0.77][github-next-release-work], we'll work on the items below:
-
- - New utility: Command Not Found
+ - Adopted XamlStyler for PowerToys Run source code. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
```

Comment:
```suggestion
```

Not closed yet since I am waiting for ColorPicker UI refresh to be merged.
Moved to 0.78 project.

Reply:
https://github.com/microsoft/PowerToys/pull/30011/files was merged right? Si tu was adopted for PowerToys Run.
@davidegiacometti , what am I missing here?

Reply:
My bad 😅 The statement was related to Run but I was thinking that the issue isn't fully closed.